api/types/types.proto

```diff
@@ -1543,3 +1542,40 @@ message DeleteWebTokenRequest {
     // Token specifies the token to delete.
     string Token = 2 [ (gogoproto.jsontag) = "token" ];
 }
+
+// EnvironmentVariable describes an environment variable with a key and value.
+message EnvironmentVariable {
+    // The name of the variable.
+    string Key = 1;
```

Comment:
nit: w/o json tags these will be serialized as-is - e.g. Capitalized. Maybe something to add to stay consistent with the rest of the attributes.